### PR TITLE
Fix table's within tables bug

### PR DESCRIPTION
**Pull request by ithielnor**
Nested tables inside the floated table header were having odd issues. These changes force floatThead to use only the thead/tbody belonging directly to the selected table(s). 

Bug: http://jsfiddle.net/z9t1wmg8/4/

In this example, the size-row gets inserted in the wrong table.

### CODE REVIEW

File: jquery.floatThead.js [hunk 1]
```diff
@@ -420,8 +420,8 @@
           }
           $table.css(layoutFixed);
           $floatTable.css(layoutFixed);
+          $header.after($newHeader);
```
Comment (mkoryak) [inline jquery.floatThead.js]: why was this change requires?

Comment (ithielnor) [inline jquery.floatThead.js]: For some reason

```
$tbody.before($newHeader);
```

Was not working at all in this two table scenario. I could not figure it out, so I tried this (which should be the same outcome) and it worked.

Comment (mkoryak) [inline jquery.floatThead.js]: Ill need to find out the exact reason why this change is required before i merge this

**Comment by ithielnor** [comment]
I created a clean test without that change and it seems to work now. It must have been something in my specific use case in my other project. 

I've worked around it differently on my project, but it seems to me the `$tbody` variable is not necessary.

Regardless, here's just the `children` changes.

**Comment by mkoryak** [comment]
Thank you!
